client/js/components/statement/publicStatementModal/formGroups/FormGroupEvaluationMailViaEmail.vue

```diff
@@ -8,7 +8,9 @@
 </license>
 
 <template>
-  <div :class="[statement.r_getFeedback === 'on' ? prefixClass('bg-color--grey-light-2') : '', prefixClass('c-statement__formblock')]">
+  <div 
+    v-if="publicParticipationFeedbackEnabled"
+    :class="[statement.r_getFeedback === 'on' ? prefixClass('bg-color--grey-light-2') : '', prefixClass('c-statement__formblock')]">
```

Comment:
maybe it could be simplified/refactored

```suggestion
    :class="[
      prefixClass('c-statement__formblock'),
      { [prefixClass('bg-color--grey-light-2')]: statement.r_getFeedback === 'on' }
    ]"
  ```